app/src/main/java/com/nervousfish/nervousfish/activities/WaitActivity.java

```diff
@@ -79,10 +79,10 @@ protected void onStart() {
      */
     @Override
     protected void onStop() {
+        super.onStop();
```

Comment:
onStop must be at the end of the method, otherwise the activity is stopped before the code below is executed (at least, on older Android versions)